### PR TITLE
Feature/nlrfim 100

### PR DESCRIPTION
**Pull request by sihpe**
Der mangler stadig idp og discovery folderne.

### CODE REVIEW

File: oiosaml/src/test/java/dk/itst/oiosaml/sp/model/OIOLogoutRequestTest.java [hunk 1]
```diff
@@ -74,29 +70,37 @@ public void testValidateLogoutRequest() throws Exception {
...
+			assert parm != null;
+			lh.validateRequest(URLDecoder.decode(parm, "UTF-8"), url.substring(url.indexOf('?') + 1), credential.getPublicKey(),location, "issuer");
 			fail();
 		} catch (LogoutRequestValidationException e) {
 			assertEquals(1, e.getErrors().size());
 		}
-		
-		lr.setNotOnOrAfter(new DateTime().minusMinutes(1));
+
+		lr.setNotOnOrAfter(new DateTime().minusHours(1));
```
Comment (mdbech) [inline oiosaml/src/test/java/dk/itst/oiosaml/sp/model/OIOLogoutRequestTest.java:90]: Check with profile, an hour is too much.

File: oiosaml/src/test/java/dk/itst/oiosaml/sp/service/TestHelper.java [hunk 2]
```diff
@@ -164,6 +164,7 @@ public static Assertion buildAssertion(String recipient, String audience) {
 		
 		assertion.setConditions(SAMLUtil.createAudienceCondition(audience));
 		assertion.getConditions().setNotOnOrAfter(new DateTime().plus(10000));
+		assertion.getConditions().setNotBefore(new DateTime().minusDays(1));
```
Comment (mdbech) [inline oiosaml/src/test/java/dk/itst/oiosaml/sp/service/TestHelper.java:167]: Usually 1 minute, but check with profile.

File: oiosaml/pom.xml [hunk 3]
```diff
@@ -44,33 +44,33 @@
 				<javadoc.opts>-Xdoclint:none</javadoc.opts>
 			</properties>
 		</profile>
-                <profile>
```
Comment (mdbech) [inline oiosaml/pom.xml:47]: SCM should point to GitHub instead.

**Review by mdbech — CHANGES_REQUESTED** [review]

File: oiosaml/src/test/java/dk/itst/oiosaml/sp/model/OIOLogoutRequestTest.java (hunk 1 above)
Comment (sihpe) [reply oiosaml/src/test/java/dk/itst/oiosaml/sp/model/OIOLogoutRequestTest.java:90]: Yes, default er 5 minutter, så det hjalp at rette det til 5
```
/**
	 * Set this value to modify the clockskew (default is 5 minutes)
	 */
	static final String PROP_CLOCK_SKEW = "oiosaml-sp.clock.skew";
```

**Review by mdbech — APPROVED** [review]

**Pull request merged**